dnsmasq.sls

```diff
@@ -24,8 +24,18 @@ dnsmasq:
 #
 # dnsmasq: failed to create listening socket for 172.17.0.1: Cannot assign requested address
 # dnsmasq: unknown interface docker0
-/lib/systemd/system/dnsmasq.service:
-  file.line:
-    - mode: ensure
-    - after: {{ '[Unit]' | regex_escape }}
-    - content: After=docker.service
+{% set dnsmasq_override_file = '/etc/systemd/system/dnsmasq.service.d/override.conf' %}
```

Comment:
I think instead of `override.conf`, a more sounding name like `docker-dependency.conf` or `after-docker.conf` is a bit nicer?

Reply:
I changed the name.